Packs/FeedLOLBAS/Integrations/FeedLOLBAS/FeedLOLBAS.py

```diff
@@ -0,0 +1,245 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+import urllib3
+from typing import Dict, Any, Tuple, List
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+''' CONSTANTS '''
+
+BASE_URL = 'https://lolbas-project.github.io/api'
```

Comment:
Should I expose the base_url to the user or just hide it?

Reply:
yes, put it as a default value of a base_url param. then if the base URL is changed for some reason, users can handle this themselves instead of fixing in the code.